### DISCUSSION TITLE
Package size

### DISCUSSION THREAD

**Discussion by will-stone** [gregberge/twc#38] [General]
Hi 👋🙂

This looks like a great utility for working with Tailwind atom like components.

I’m just wondering about the 0.49kb size. From looking at the source, the lib depends on radix-ui/slot which is 906b: https://www.radix-ui.com/primitives/docs/utilities/slot

**Comment by gregberge** [comment]
Yeah you are right, but I think lot of codebase already have this utility, that's why I didn't include it. Same thing for clsx.

**Reply by will-stone** [reply]
It’s more of a case that I don’t think that’s how size is counted. Imagine saying that your app is only 4kb because you aren’t counting the 1mb of dependencies. And when the package gets picked up by Bundlephobia someone else will probably raise this point.

Maybe you could say “only 0.49kb + 2 dependencies bringing the total to 1.5k” (I don’t know what the total is, this is a guess). 

Anyway, I’m not sure you’ll find many projects stating total size but excluding deps.

**Reply by gregberge** [reply]
Yeah you are right.
